docs/recipes/RemoveMobxStateTree.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Remove Mobx-State-Tree
+description: How to remove Mobx-State-Tree from an Ignite project
+tags:
+  - MobX
+  - State Management
+last_update:
+  author: Justin Poliachik
+publish_date: 2024-02-05
+---
+
+# Remove Mobx-State-Tree
+
+By default, Ignite uses [Mobx-State-Tree](https://mobx-state-tree.js.org/) as the default state management solution. While we love [Mobx-State-Tree at Infinite Red](https://docs.infinite.red/ignite-cli/concept/MobX-State-Tree/), we understand the landscape is rich with great alternatives that you may want to use instead.
+
+This guide will show you how to remove Mobx-State-Tree from an Ignite-generated project and get to a "blank slate" with no state management at all.
+
+## Steps
+
+1. Let's start by removing all Mobx-related dependencies from `package.json`, then run `yarn` or `npm i`
+
+**package.json**
+
+```diff
+--"mobx": "6.10.2",
+--"mobx-react-lite": "4.0.5",
+--"mobx-state-tree": "5.3.0",
+
+--"reactotron-mst": "3.1.5",
+```
+
+2. Ignite places all Mobx-State-Tree models in the `models/`. Remove this entire directory and all files within it, these are not needed anymore.
+
+```terminal
+rm -rf ./app/models
+```
+
+:::note
+If you are migrating a project with several existing models, you may want to keep a copy of these around for reference as you migrate to your new system.
+:::
+
+3. Remove the `reactotron-mst` plugin from Reactotron's config
+
+**devtools/ReactotronConfig.ts**
+
+```diff
+--import { mst } from "reactotron-mst"
+
+...
+
+const reactotron = Reactotron.configure({
+  name: require("../../package.json").name,
+  onConnect: () => {
+    /** since this file gets hot reloaded, let's clear the past logs every time we connect */
+    Reactotron.clear()
+  },
+--}).use(
+--  mst({
+--    /** ignore some chatty `mobx-state-tree` actions  */
+--    filter: (event) => /postProcessSnapshot|@APPLY_SNAPSHOT/.test(event.name) === false,
+--  }),
+--)
+++})
+```
+
+4. Remove `observer()` wrapped components and reformat as functional React components
+
+- Do a project-wide search for `observer(` and replace each component instance with the following pattern:
```

Comment:
Even though this is one example of the several places to remove observer, I would put your bold text like you've been doing above snippets to just indicate to the reader which file you're modifying in the event s/he is lost

```
**app/screens/WelcomeScreen.tsx**
```